cmd/gomfweb/main.go

```diff
@@ -87,44 +87,71 @@ func index(w http.ResponseWriter, r *http.Request) {
+
+  <title>Go Microformats Parser</title>
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/css/bootstrap.min.css" integrity="sha384-rwoIResjU2yc3z8GV/NPeZWAv56rSmLldC3R/AZzGRnGxQQKnKkoFVhFQhNUwEyJ" crossorigin="anonymous">
+  <style>
+    form label { font-weight: bold; }
+    form textarea, form input[type=url] { font-family: "SF Mono", Menlo, Monaco, Consolas, "Liberation Mono", "Courier New", monospace; }
+    form .form-control:disabled { cursor: default; background: #efefef; color: black; }
+  </style>
 </head>
+
 <body>
-  <h1>go microformats parser</h1>
-  <h2>Parse a URL</h2>
-  <form method="GET">
-    <input name="url" type="url" placeholder="https://indieweb.org/" />
-    <input type="submit" value="Parse" />
-  </form>
-
-  <h2>Parse HTML</h2>
-  <form method="POST">
-    <label for="html">HTML</label>
-    <textarea id="html" name="html" rows="15">{{ .HTML }}</textarea>
-    <label for="url">Base URL</label>
-    <input id="url" name="url" type="url" value="{{ .URL }}" placeholder="https://indieweb.org/" />
-    <input type="submit" value="Parse"/>
-  </form><br>
-
-{{ with .JSON }}
-  <h2>JSON</h2>
-  <pre><code>{{ . }}
-</code></pre>
-{{ end }}
-<ul>
-  <li><a href="http://microformats.org/wiki/about">About microformats</a></li>
-  <li><a href="https://github.com/willnorris/microformats/tree/master/cmd/gomfweb">Source code for this site</a></li>
-  <li><a href="http://microformats.org/wiki/microformats2#Parsers">Other microformats parsers</a></li>
-</ul>
+  <main class="container">
+    <h1 class="mt-5 mb-3">Microformats Parser (Go)</h1>
+
+    <form method="get">
+      <div class="form-group">
+        <label for"url">Enter a URL</label>
+        <input name="url" type="url" placeholder="https://indieweb.org" class="form-control form-control-lg" />
+      </div>
+
+      <button type="submit" class="btn btn-lg btn-success">Parse</button>
+    </form>
+
+    <h2 class="h4 my-5">OR parse just a snippet of HTML</h2>
+
+    <form method="post" class="mb-5">
+      <div class="form-group">
+        <label for="html">HTML</label>
+        <textarea id="html" name="html" rows="6" class="form-control form-control-lg">{{ .HTML }}</textarea>
+      </div>
+
+      <div class="form-group">
+        <label for="base-url">Base URL</label>
+        <input id="base-url" name="base-url" type="url" value="{{ .URL }}" placeholder="https://indieweb.org" class="form-control form-control-lg" />
+      </div>
+
+      <button type="submit" class="btn btn-lg btn-success">Parse</button>
+    </form>
+
+    {{ with .JSON }}
+    <div class="form-group mb-5">
+      <label for="json">JSON</label>
+      <textarea id="json" name="json" rows="10" class="form-control form-control-lg" disabled="disabled">{{ . }}</textarea>
+    </div>
+    {{ end }}
+
+    <footer class="mb-5">
+      <ul>
+        <li><a href="https://microformats.io">About Microformats</a></li>
```

Comment:
"fix microformats.org" is non-trivial. Partly because of cooks in the kitchen, partly because of tech stack (old media wiki install), partly because long ago the decision was made to put the blog on the homepage. I'm happy to change the About MF link to mf.org if that's a blocker for you to merge this or at a time when mf.org is "fixed".